xray/src/build_quadtree.rs

```diff
@@ -142,7 +142,7 @@ pub fn run<T: Extension>(data_provider_factory: DataProviderFactory) {
             xray => ColoringStrategyKind::XRay,
             colored => ColoringStrategyKind::Colored(binning),
             colored_with_intensity => ColoringStrategyKind::ColoredWithIntensity(
-                value_t!(args, "min_intensity", f32).unwrap_or(1.),
+                value_t!(args, "min_intensity", f32).unwrap_or(0.),
                 value_t!(args, "max_intensity", f32).unwrap_or(1.),
```

Comment:
Wouldn't it make more sense to panic if provided arguments cannot be parsed?

Reply:
Problably, this was just some random stuff I observed... how about just removing the `_or()` part  and make it an `unwrap()`?

Reply:
Hm, actually it makes most sense to provide default values above and change it here to `expect`s

Reply:
:+1: that makes sense.

Reply:
Done, PTAL!

Reply:
lgtm